src/modules/components/asset/assetTransfer/assetTransferAddress/assetTransferAddress.test.tsx

```diff
@@ -0,0 +1,69 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import { OdsModulesProvider } from '../../../odsModulesProvider';
+import { AssetTransferAddress, type IAssetTransferAddressProps } from './assetTransferAddress';
+
+jest.mock('../../../member/memberAvatar', () => ({ MemberAvatar: () => <div data-testid="member-avatar-mock" /> }));
```

Comment:
By using the index file we can change the internal structure on the component without affecting eventual imports, let's always import/mock from the index files
```suggestion
jest.mock('../../../member', () => ({ MemberAvatar: () => <div data-testid="member-avatar-mock" /> }));
```